src/Test/Unit/Service/ElasticSearchTest.php

```diff
@@ -58,13 +59,17 @@ public function setUp()
         );
     }
 
-    public function testGetVersionElasticSearchNotExistInRelationShips()
+    /**
+     * @throws ServiceException
+     */
+    public function testGetVersionElasticSearchNotExistInRelationShips(): void
```

Comment:
RelationShips SHOULD BE Relationships
(Or is that being too picky?)

Reply:
Thanks, fixed